dali/python/backend_impl.cc

```diff
@@ -111,8 +112,82 @@ py::dict ArrayInterfaceRepr(Tensor<Backend> &t) {
   return d;
 }
 
+template<typename SrcBackend>
+TensorListShape<> CreateShape(TensorShape<> &shape, TensorList<SrcBackend>*) {
+  std::vector<Index> tensor_shape(shape.size()-1);
+  for (int i = 1; i < shape.size(); ++i) {
+    tensor_shape[i-1] = shape[i];
+  }
+  return uniform_list_shape(shape[0], tensor_shape);
+}
+
+template<typename SrcBackend>
+TensorShape<> CreateShape(TensorShape<> &shape, Tensor<SrcBackend>*) {
+  return shape;
+}
+
+template<typename TStrides, typename TShape>
+void CheckStrides(TStrides &strides, TShape &shape, size_t type_size,
+                  size_t strides_size, size_t shape_size) {
```

Comment:
```suggestion
void CheckContiguousTensor(const TStrides &strides, size_t num_strides, const TShape &shape, 
                  size_t num_extents, size_t element_size) {
```
...and add an overload:
```
void CheckContiguousTensor(const TStrides &strides, const TShape &shape, size_t element_size) {
  CheckContiguousTensor(strides, dali::size(strides), shape, dali::size(shape), element_size);
}
```

Reply:
Done